requirements/constraints.txt

```diff
@@ -14,3 +14,9 @@ futures ; python_version == "2.7"
 # A dependency of pytest.  Pytest doesn't constrain it and 6.0.0 onward
 # only works with python 3
 more-itertools<6.0.0
+
+# Newer versions dropped support for Python 2.7
+edx-lint<1.5.0
+
+# Newer versions dropped support for Python 2.7
+tox-battery<0.6.0
```

Comment:
Could you please add a couple of tickets under the code ownership epic to drop Python 2.7 support and add Python 3.8 support in this repo?  Part of the 2.7 ticket would be confirming that we actually don't need the 2.7 support for anything now.

Reply:
Sure. I'll make the JIRA tickets. meanwhile can you merge this PR? I don't have merge access.